packages/field/src/LionField.js

```diff
@@ -136,60 +123,24 @@ export class LionField extends FormControlMixin(
     this.value = ''; // can't set null here, because IE11 treats it as a string
   }
 
-  /* * * * * * * * * *
-    Event Handlers */
-
+  // Needed for choice-inputs and select (IE11) as an alternative to the input event.
+  // TODO: add in FormatMixin and remove here and in lion-select
   _onChange() {
     if (super._onChange) super._onChange();
     this.dispatchEvent(
       new CustomEvent('user-input-changed', {
         bubbles: true,
+        composed: true,
```

Comment:
previously this bubbled, meaning that it will reach all parent elements in the same shadow root where the host (custom input) is located, so for forms and custom forms inputs which are entirely located in the same shadow root (by our convention) it was enough

what changed? do we need some elements extending from LionField to be in a deeper shadow root than the rest of the form? can you show the code you are targeting, like a DOM structure with all elements and shadow roots, roughly?

Reply:
We fire same event in different places (with composed), there is no reason to do it without in this particular case. Or more generic, there is no reason to be inconsistent throughout our code

Reply:
To simplifiy this merge request, I will put this back in original file (and state). In a later refactor, we can move this logic to one single, appropriate place with shared functionality for ChoiceInput and Select